yugabyte/src/yugabyte/bank_improved.clj

```diff
@@ -13,12 +13,10 @@
   for YSQL (update-insert-delete) deleted key will be chosen from contention-keys"
   (:refer-clojure :exclude
                   [test])
-  (:require [clojure [pprint :refer [pprint]]]
-            [clojure.tools.logging :refer [info]]
-            [jepsen.tests.bank :as bank]
-            [knossos.op :as op]
+  (:require [jepsen.tests.bank :as bank]
             [clojure.core.reducers :as r]
             [jepsen
+             [history :as h]
```

Comment:
Interesting, I don't see `history.clj` in `jepsen` - but I see it used all over Jepsen

Reply:
It's separated package now https://github.com/jepsen-io/history